task/02-numbers-tasks.js

```diff
@@ -54,7 +52,11 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-    throw new Error('Not implemented');
+  if (value1 + value2 === 0) return 0;
+  if (value1 + value2 > Number.MAX_VALUE) {
+    return (BigInt(value1) + BigInt(value2)) / 2n;
+  }
+  return (value1 + value2) / 2;
```

Comment:
Add a newline before the return statement, this is a good practice according to the [following ESLint rule](https://eslint.org/docs/latest/rules/newline-before-return)